src/qibolab/platforms/multiqubit.py

```diff
@@ -58,6 +58,32 @@ def set_lo_readout_frequency(self, qubit, freq):
     def get_lo_readout_frequency(self, qubit):
         return self.ro_port[qubit].lo_frequency
 
+    def set_lo_twpa_frequency(self, qubit, freq):
+        for instrument in self.instruments:
+            if "twpa" in instrument:
+                self.instruments[instrument].frequency = freq
+                return None
+        raise_error(NotImplementedError, "No twpa instrument found in the platform. ")
+
+    def get_lo_twpa_frequency(self, qubit):
+        for instrument in self.instruments:
+            if "twpa" in instrument:
+                return self.instruments[instrument].frequency
+        raise_error(NotImplementedError, "No twpa instrument found in the platform. ")
+
+    def set_lo_twpa_power(self, qubit, power):
+        for instrument in self.instruments:
+            if "twpa" in instrument:
+                self.instruments[instrument].power = power
+                return None
```

Comment:
Several things can go wrong here, such as:

* `qubit` argument is not used so you don't know if you are changing the TWPA affecting the given qubit,
* `return` will exit the function and only update the first instrument named as `"twpa"` in the `self.instruments` list,
* it is assumed that the name of the instrument you want to update contains the word `"twpa"`

however as you say in the first post this is works only when we have a single TWPA, so these points don't matter, except maybe the third.

I am not familiar with the `MultiqubitPlatform` to say if these issues can be easily fixed, so I am fine with merging as it is.